zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py

```diff
@@ -21,47 +21,56 @@ class WikipediaHandler(object):
 
     META = {
         'name': 'Wikipedia',
-        'description': 'Searches Wikipedia for a term and returns the top article.',
+        'description': 'Searches Wikipedia for a term and returns the top 3 articles.',
     }
 
     def usage(self):
         return '''
             This plugin will allow users to directly search
-            Wikipedia for a specific key term and get the top
-            article that is returned from the search. Users
+            Wikipedia for a specific key term and get the top 3
+            articles that is returned from the search. Users
             should preface searches with "@mention-bot".
-            '''
+            @mention-bot <name of article>'''
 
     def handle_message(self, message, bot_handler):
         bot_response = self.get_bot_wiki_response(message, bot_handler)
         bot_handler.send_reply(message, bot_response)
 
     def get_bot_wiki_response(self, message, bot_handler):
-        help_text = 'Please enter your message after @mention-bot'
+        '''This function returns the URLs of the requested topic.'''
+
+        help_text = 'Please enter your search term after @mention-bot'
+
+        # Checking if the link exists.
         query = message['content']
         if query == '':
             return help_text
+
         query_wiki_link = ('https://en.wikipedia.org/w/api.php?action=query&'
                            'list=search&srsearch=%s&format=json'
                            % (urllib.parse.quote(query),))
+        print(query_wiki_link)
         try:
             data = requests.get(query_wiki_link)
+
         except requests.exceptions.RequestException:
             logging.error('broken link')
             return
 
+        # Checking if the bot accessed the link.
         if data.status_code != 200:
-            logging.error('unsuccessful data')
+            logging.error('Page not found.')
```

Comment:
This one-line change could be its own commit.

Reply:
Can you PM me on the chat.zulip.org,
I can really use some help on the commits :P